CHANGELOG.md

```diff
@@ -12,6 +12,18 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 * py.typed added, type checking active on PyGraphistry!
 * transform() and transform_umap() now require some parameters to be keyword-only
 
+### Added
+
+* GFQL: Comparison predicates (`gt`, `lt`, `ge`, `le`, `eq`, `ne`, `between`) and `is_in` now support datetime, date, and time values with timezone awareness
+
+### Changed
+
+* GFQL: Temporal value classes (`DateTimeValue`, `DateValue`, `TimeValue`) are now exported from `graphistry.compute`
+
+### Internal
+
+* Remove unused imports across predicate modules
```

Comment:
github ui is showing 1243 removals on this changelog, that is pretty messed up. Redo how we're doing changes on this file so not happening.